### PR TITLE
spelling error and 'vector' ambiguous symbol

### PR DESCRIPTION
**Pull request by xwnb**
a spelling error
Compiler Error C2872: 'vector' : ambiguous symbol in vs 2019

### CODE REVIEW

File: docs/image_apis.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ int getStereoAndDepthImages()
     msr::airlib::MultirotorRpcLibClient client;
 
     //get right, left and depth images. First two as png, second as float16.
-    vector<ImageRequest> request = { 
+    msr::airlib::vector<ImageRequest> request = {
```
Comment (rajat2004) [inline docs/image_apis.md:109]: I think this should be `std`, don't think AirSim defines it's own vector

Comment (xwnb) [reply docs/image_apis.md:109]: Agree. In the file Common.hpp, it has "using vector = std::vector<T>", so std::vector or msr::airlib::vector both work. However, it occurs  Compiler Error C2872 if missing a namespace.

Comment (xwnb) [reply docs/image_apis.md:109]: Agree. In the file _Common.hpp_, it has "_using vector = std::vector<T>_", so both _std::vector_ or _msr::airlib::vector_ work. However, it occurs  **Compiler Error C2872** if missing a namespace.

Comment (rajat2004) [reply docs/image_apis.md:109]: That makes sense, I think then the problem is that we have 2 namespaces here, we should probably remove `using namespace std;` from the block, see [HelloDrone](https://github.com/microsoft/AirSim/blob/master/HelloDrone/main.cpp) or [HelloCar](https://github.com/microsoft/AirSim/blob/master/HelloCar/main.cpp)

**Pull request merged**